components/test_raftstore/src/util.rs

```diff
@@ -462,7 +462,7 @@ pub fn must_read_on_peer<T: Simulator>(
     key: &[u8],
     value: &[u8],
 ) {
-    let timeout = Duration::from_secs(1);
+    let timeout = Duration::from_secs(5);
```

Comment:
Which test does it fix?

Reply:
For example `test_lease_unsafe_during_leader_transfers`.